nixnet/types.py

```diff
@@ -220,3 +221,110 @@ def __repr__(self):
             self.echo,
             self.type,
             self.timestamp)
+
+
+class LinFrame(object):
+    """LIN Frame.
+
+    Attributes:
+        identifier: An integer representing the LIN frame arbitration identifier.
+        echo: A boolean indicating if the frame is an echo of a successful
+            transmit rather than being received from the network.
+        type: Frame type as described in :any:`nixnet._enums.FrameType`.
+        timestamp: Absolute time the XNET interface received the end-of-frame.
+        payload: A byte string representing the payload.
+        eventid: Identifier for an event-triggered slot.
+        eventslot: A boolean indicating whether the frame was received within an
+            event-triggered slot or an unconditional or sporadic slot.
+    """
+
+    __slots__ = [
+        "identifier",
+        "echo",
+        "type",
+        "timestamp",
+        "eventid",
+        "eventslot",
+        "payload"]
```

Comment:
Reading the documentation, it seems like `eventslot` impacts the meaning of `identifier` and `eventid`

We should at least re-order it so `eventslot` is always referenced before `eventsid` to communicate this.  I'd suggest putting it before `identifier` but that'll mess up `__init__`s defaulting of parameters.

http://zone.ni.com/reference/en-XX/help/372841L-01/nixnet/rawframeformat_c/